File: tests/human/x/go/if_then_else_nested.go
//go:build ignore

package main

import "fmt"

func main() {
	x := 8
	var msg string
	if x > 10 {
		msg = "big"
	} else if x > 5 {
		msg = "medium"
	} else {
		msg = "small"
	}
	fmt.Println(msg)
}

File: tests/machine/x/pl/if_then_else_nested.pl
:- style_check(-singleton).
:- initialization(main, main).
main :-
    X is 8,
    ((X @> 5) -> _V0 = "medium" ; _V0 = "small"),
    ((X @> 10) -> _V1 = "big" ; _V1 = _V0),
    Msg = _V1,
    writeln(Msg),
    true.
